.gitignore

```diff
@@ -113,3 +113,6 @@ desktop.ini
 
 # monkeytype
 monkeytype.sqlite3
+
+# Include
+/include
```

Comment:
What is this for? I can't find an explanation for this addition in the comments, and even then this should be in a separate PR.